Tests/SwiftPrometheusTests/PrometheusMetricsTests.swift

```diff
@@ -1,27 +1,37 @@
 import XCTest
+import NIO
 @testable import Prometheus
 @testable import CoreMetrics
 
 final class PrometheusMetricsTests: XCTestCase {
     
     var prom: PrometheusClient!
+    var group: EventLoopGroup!
+    var elg: EventLoop {
```

Comment:
```suggestion
    var eventLoop: EventLoop {
```

Since it's not an `EventLoopGroup`